Correct grammatical errors in README.md (#20509)
PowerShell/PowerShell: README.md

Welcome to the PowerShell GitHub Community!
[PowerShell](https://learn.microsoft.com/powershell/scripting/overview) is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized

> for dealing with structured data (e.g., JSON, CSV, XML, etc.), REST APIs, and object models.

Reviewer: ```suggestion
for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models.
```

It includes a command-line shell, an associated scripting language, and a framework for processing cmdlets.

[logo]: https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/ps_black_64.svg?sanitize=true

## Windows PowerShell vs. PowerShell Core

> Although this repository started as a fork of the Windows PowerShell codebase, changes made in this repository do not automatically make their way back to Windows PowerShell 5.1.

Reviewer: ```suggestion
Although this repository started as a fork of the Windows PowerShell codebase, changes made in this repository are not automatically ported back to Windows PowerShell 5.1.
```
